egs2/tedlium2/asr1/local/data.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+# Copyright 2017 Johns Hopkins University (Shinji Watanabe)
+#  Apache 2.0  (http://www.apache.org/licenses/LICENSE-2.0)
+# Modified by flylili001
+
+. ./path.sh || exit 1;
+. ./cmd.sh || exit 1;
+
+# general configuration
+stage=-1       # start from -1 if you need to start from data download
+stop_stage=100
+
```

Comment:
please add `. utils/parse_options.sh || exit 1;` to allow argument parsing here.